backend/app/core/oauth.py

```
@@ -0,0 +1,309 @@
+            redirect_uri=settings.OAUTH_REDIRECT_URL,
+            scope="openid email profile"
+        )
+
+        authorization_url, _ = client.create_authorization_url(
+            self.authorization_endpoint,
+            state=state
+        )
+
+        return authorization_url
+
+    async def get_user_info(self, code: str, state: str) -> Dict[str, Any]:
```

Ruff (0.13.1)

Unused method argument: `state`

(ARG002)